my-sms/app/controllers/students_controller.rb

```diff
@@ -84,4 +87,10 @@ def destroy
   def find_student
     @student = Student.find(params[:id])
   end
+
+  def flash_errors
+    student.errors.full_messages.each do |error|
```

Comment:
should this be `@student` instead?

```suggestion
    @student.errors.full_messages.each do |error|
```

Reply:
Fixed in 19b82cd3dc7de83283b7660a0c09c3b44eb187c8